models/issue.go

```diff
@@ -753,6 +753,24 @@ func (issue *Issue) ChangeRef(doer *User, oldRef string) (err error) {
 		return fmt.Errorf("updateIssueCols: %v", err)
 	}
 
+	if err = issue.loadRepo(db.GetEngine(ctx)); err != nil {
+		return fmt.Errorf("loadRepo: %v", err)
+	}
+	oldRef = strings.TrimPrefix(oldRef, "refs/heads/")
+	newRef := strings.TrimPrefix(issue.Ref, "refs/heads/")
+
+	opts := &CreateCommentOptions{
+		Type:   CommentTypeChangeIssueRef,
+		Doer:   doer,
+		Repo:   issue.Repo,
+		Issue:  issue,
+		OldRef: oldRef,
+		NewRef: newRef,
```

Comment:
```suggestion
	oldRefFriendly := strings.TrimPrefix(oldRef, "refs/heads/")
	newRefFriendly := strings.TrimPrefix(issue.Ref, "refs/heads/")

	opts := &CreateCommentOptions{
		Type:   CommentTypeChangeIssueRef,
		Doer:   doer,
		Repo:   issue.Repo,
		Issue:  issue,
		OldRef: oldRefFriendly,
		NewRef: newRefFriendly,
```

It's better not to change the meaning of a variable. Otherwise it may lead to bugs if other developers access to the changed variable later.

ps: I haven't checked the real meaning of OldRef and NewRef in CreateCommentOptions. If they are free to be used to store anything thing changed, then that's OK.